split/resource_split_split.go

```diff
@@ -50,6 +50,15 @@ func resourceSplitSplit() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+
+			"tags":{
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional: true, 
+				ForceNew: true,
```

Comment:
Is `ForceNew: true` needed here? It does seem you can update the feature flag without needing to recreate the feature flag itself.

Reply:
You cant actually remove tags from a feature flag, only add new ones. At least that's how it was when i tested it.